presto-main/src/main/java/com/facebook/presto/server/security/OpenLdapBinder.java

```diff
@@ -14,19 +14,20 @@
 package com.facebook.presto.server.security;
 
 import javax.inject.Inject;
-import javax.naming.directory.DirContext;
 
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
 public class OpenLdapBinder
         implements LdapBinder
 {
+    private final String userObjectClass;
     private final String baseDistinguishedName;
 
     @Inject
     public OpenLdapBinder(LdapServerConfig config)
     {
+        userObjectClass = requireNonNull(config.getUserObjectClass(), "userObjectClass is null");
```

Comment:
`userObjectClass` will be `person` by default. So if someone forgets to set `authentication.ldap.user-object-class` property for OpenLDAP (which is possible since this property is not mandatory?), would this break the code? 
CC: @anusudarsan